CLR/Core/CLR_RT_HeapBlock.cpp

```diff
@@ -1897,21 +1897,6 @@ HRESULT CLR_RT_HeapBlock::TransparentProxyValidate() const
 #endif //TINYCLR_APPDOMAINS
 ////////////////////////////////////////////////////////////////////////////////////////////////////
 
-#if defined(PLATFORM_WINDOWS) || defined(PLATFORM_WINCE)
-
-static void MyAssert()
-{
-    NATIVE_PROFILE_CLR_CORE();
-    DebugBreak();
-}
-
-#else
-
-static void MyAssert() {}
-
-#endif

```

Comment:
Nit-pick: In general, we should tag #else/#endif with the original condition.  e.g. "#endif // defined(PLATFORM_WINDOWS) || defined(PLATFORM_WINCE)"